lib/src/map/map_state_widget.dart

```diff
@@ -12,10 +13,10 @@ class MapStateInheritedWidget extends InheritedWidget {
 
   @override
   bool updateShouldNotify(MapStateInheritedWidget oldWidget) {
-    return oldWidget.mapState.zoom == mapState.zoom &&
-        oldWidget.mapState.center == mapState.center &&
-        oldWidget.mapState.bounds == mapState.bounds &&
-        oldWidget.mapState.rotation == mapState.rotation;
+    // mapState will be the same because FlutterMapState create MapState object just once
+    // and pass the same instance to the old / new MapStateInheritedWidget
+    // Moreover MapStateInheritedWidget isn't cached so all of it's child will be updated no matter if we return here with false
+    return true;
```

Comment:
The rewrote version should be:
```dart
  @override
  bool updateShouldNotify(MapStateInheritedWidget oldWidget) {
    return oldWidget.mapState.zoom != mapState.zoom ||
        oldWidget.mapState.center != mapState.center ||
        oldWidget.mapState.bounds != mapState.bounds ||
        oldWidget.mapState.rotation != mapState.rotation;
  }
```
Since mapState will be always same instance it is safe to return true
Moreover MapStateInheritedWidget **child** isn't cached so all of it's child will be updated no matter if we return here with false